mnamer/target.py

```diff
@@ -133,6 +133,10 @@ def parse(path: str, media: str) -> Metadata:
                     meta["series"] += " (%d)" % data["year"]
             if "season" in data:
                 meta["season"] = str(data["season"])
+                if meta["series"] in season_overrides:
```

Comment:
This is breaking tests. `"series"` is not guaranteed to be a key of `meta`. The check would probably need to be something like `if "series" in meta and meta["series"] in season_overrides`.